src/adapters/chartjs.js

```diff
@@ -543,7 +543,7 @@ export default class {
     this.library = library;
   }
 
-  renderLineChart(chart, chartType) {
+  renderLineChartFromChartjs(chart, chartType) {
```

Comment:
are we still using charts from ChartJs? I can see function name render LineChart from ChartJS? can you explain?

Reply:
no we are not, there was already a linechart in there created by you, i just changed the name cuz chart js adapter is higher up and it starts running stuff from your chartjs first